packages/nx-flutter/src/generators/project/lib/prompt-additional-options.ts

```diff
@@ -2,12 +2,18 @@ import { Tree } from '@nrwl/devkit';
 import { prompt } from 'enquirer';
 import { AndroidLanguageType, IosLanguageType, NormalizedSchema, PlatformType } from '../schema';
 
-type PromptResultType = { platforms: PlatformType[], androidLanguage?: AndroidLanguageType, iosLanguage?: IosLanguageType };
+type PromptResultType = { platforms: PlatformType[], androidLanguage?: AndroidLanguageType, iosLanguage?: IosLanguageType, fvm: boolean };
 
 export async function promptAdditionalOptions(tree: Tree, options: NormalizedSchema) {
     if (process.env.NX_INTERACTIVE === 'true') {
 
-        await prompt([{
+        await prompt([
+          {
+            name: 'fvm',
```

Comment:
Can you remove the prompt here, and add it to the  `generators/project/schema.json`?

This section is for dynamic/complex prompts that can not be expressed via the schema.json API

Reply:
Absolutely! @tinesoft

Reply:
Hi John,

did you have time to apply the requested changes?

If not, and if you don't mind, I'll take over and continue the PR (you'll still be the author of course). 

I'm asking because I'd like to release this before the end of the week if possible.

Thanks